mrmustard/lab_dev/states/states.py

```diff
@@ -205,39 +205,23 @@ def __init__(
         n: Union[int, Sequence[int]],
         cutoffs: Optional[Union[int, Sequence[int]]] = None,
     ) -> None:
-        super().__init__(modes=modes, name="N")
-
-        self._n = math.atleast_1d(n)
-        if len(self._n) == 1:
-            self._n = math.tile(self._n, [len(modes)])
-        if len(self._n) != len(modes):
-            msg = f"Length of ``n`` must be 1 or {len(modes)}, found {len(self._n)}."
-            raise ValueError(msg)
-
-        self._cutoffs = math.atleast_1d(cutoffs) if cutoffs else self.n
-        if len(self._cutoffs) == 1:
-            self._cutoffs = math.tile(self._cutoffs, [len(modes)])
-        if len(self._cutoffs) != len(modes):
-            msg = f"Length of ``cutoffs`` must be 1 or {len(modes)}, found {len(self._cutoffs)}."
-            raise ValueError(msg)
+        super().__init__(modes=modes, name=f"{n}")
```

Comment:
I recognize this is a subjective thing, but this now looks weird to me. Why not Number(n) or Fock(n) as the name?
![image](https://github.com/user-attachments/assets/e9aa19ea-2d17-4876-842e-1804445f7b2b)

Reply:
I won't die on this hill, FYI 😆

Reply:
Oh that was so that in the circuit a photon number state looks like this:

<img width="131" alt="Screenshot 2024-07-12 at 9 37 14 AM" src="https://github.com/user-attachments/assets/dcef65cc-6424-4719-af64-c58a50115107">

what we should do is not to use that in the visualization in your screenshot